projects/opal-frontend-common/components/govuk/govuk-error-message/govuk-error-message.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { GovUkErrorMessageComponent } from './govuk-error-message.component';
+import { By } from '@angular/platform-browser';
+
+describe('GovUkErrorMessageComponent', () => {
+  let component: GovUkErrorMessageComponent;
+  let fixture: ComponentFixture<GovUkErrorMessageComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [GovUkErrorMessageComponent],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(GovUkErrorMessageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should display the error message when error is true', () => {
+    component.error = true;
+    component.errorMessage = 'This is an error message';
+    component.elementId = 'error-id';
+    fixture.detectChanges();
+
+    const errorElement = fixture.debugElement.query(By.css('.govuk-error-message'));
+    expect(errorElement).toBeTruthy();
+    expect(errorElement.nativeElement.textContent.trim()).toContain('This is an error message');
```

Comment:
Is this part needed? errorElement.nativeElement.textContent.trim()

I think the below should work?
```suggestion
    expect(errorElement.nativeElement.textContent).toContain('This is an error message');
```